src/database/schema/pool-balance.ts

```diff
@@ -0,0 +1,44 @@
+import { pgTable, timestamp, varchar, integer, unique, serial, char } from 'drizzle-orm/pg-core';
+
+import { chains } from './chains';
+
+export const poolBalanceTable = pgTable(
+  'pool-balance',
+  {
+    id: serial('id').primaryKey(),
+    baseId: varchar('base_id').notNull(),
+    quoteId: varchar('quote_id').notNull(),
```

Comment:
Should log reference to pool instead of token addresses
```suggestion
    poolId: integer('pool_id')
      .notNull()
      .references(() => pools.id, { onDelete: 'cascade' }),
```